src/Microsoft.Graph/Generated/Admin/ServiceAnnouncement/HealthOverviews/Item/ServiceHealthItemRequestBuilder.cs

```diff
@@ -39,17 +39,17 @@ public class ServiceHealthItemRequestBuilder : BaseRequestBuilder {
         /// <param name="requestConfiguration">Configuration for the request such as headers, query parameters, and middleware options.</param>
 #if NETSTANDARD2_1_OR_GREATER || NETCOREAPP3_1_OR_GREATER
 #nullable enable
-        public async Task DeleteAsync(Action<ServiceHealthItemRequestBuilderDeleteRequestConfiguration>? requestConfiguration = default, CancellationToken cancellationToken = default) {
+        public async Task<Stream?> DeleteAsync(Action<ServiceHealthItemRequestBuilderDeleteRequestConfiguration>? requestConfiguration = default, CancellationToken cancellationToken = default) {
 #nullable restore
 #else
-        public async Task DeleteAsync(Action<ServiceHealthItemRequestBuilderDeleteRequestConfiguration> requestConfiguration = default, CancellationToken cancellationToken = default) {
+        public async Task<Stream> DeleteAsync(Action<ServiceHealthItemRequestBuilderDeleteRequestConfiguration> requestConfiguration = default, CancellationToken cancellationToken = default) {
```

Comment:
Possibly related to https://github.com/microsoft/OpenAPI.NET.OData/pull/357/

Any chance it was intedend to change the 204 response for DELETE methods to the 2XX range as well.  
@baywet @irvinesunday